config/example/node.yaml

```diff
@@ -91,6 +91,7 @@ morph:
 
 apiclient:
   dial_timeout: 15s  # timeout for NEOFS API client connection
+  stream_timeout: 20s
```

Comment:
Lets clarify that it's a per-message timeout for client-side streams.

Reply:
Fixed